tests/plugins/test_add_labels.py

```diff
@@ -624,30 +624,41 @@ def test_add_labels_equal_aliases2(tmpdir, docker_tasker, caplog, base_l,
             assert expected_log in caplog.text
 
 
-@pytest.mark.parametrize("parent_scope, docker_scope, result_scope, dont_overwrite", [  # noqa
-    (None, None, "restricted", False),
-    ("public", None, "restricted", False),
-    ("private", None, "restricted", False),
-    ("restricted", "public", "public", False),
-    ("restricted", "restricted", "restricted", False),
-    ("restricted", "private", "private", False),
-    (None, None, "restricted", True),
-    ("public", None, "restricted", True),
-    ("private", None, "restricted", True),
-    ("restricted", "public", "public", True),
-    ("restricted", "restricted", "restricted", True),
-    ("restricted", "private", "private", True),
-    ("public", "private", "private", True)
+@pytest.mark.parametrize("label_name, parent_val, docker_val, result_val, dont_overwrite", [  # noqa
+    ("distribution-scope", None, None, "restricted", False),
+    ("distribution-scope", "public", None, "restricted", False),
+    ("distribution-scope", "private", None, "restricted", False),
+    ("distribution-scope", "restricted", "public", "public", False),
+    ("distribution-scope", "restricted", "restricted", "restricted", False),
+    ("distribution-scope", "restricted", "private", "private", False),
+    ("distribution-scope", None, None, "restricted", True),
+    ("distribution-scope", "public", None, "restricted", True),
+    ("distribution-scope", "private", None, "restricted", True),
+    ("distribution-scope", "restricted", "public", "public", True),
+    ("distribution-scope", "restricted", "restricted", "restricted", True),
+    ("distribution-scope", "restricted", "private", "private", True),
+    ("distribution-scope", "public", "private", "private", True),
+
+    ("com.redhat.license_terms", None, None, "default_license", False),
+    ("com.redhat.license_terms", None, "special", "special", False),
+    ("com.redhat.license_terms", "parent", "special", "special", False),
+    ("com.redhat.license_terms", "parent", None, "default_license", False),
+    ("com.redhat.license_terms", None, None, "default_license", True),
+    ("com.redhat.license_terms", None, "special", "special", True),
+    ("com.redhat.license_terms", "parent", "special", "special", True),
+    ("com.redhat.license_terms", "parent", None, "default_license", True),
 ])
-def test_dont_overwrite_distribution_scope(tmpdir, docker_tasker, parent_scope,
-                                           docker_scope, result_scope, dont_overwrite,
-                                           reactor_config_map):
+def test_dont_overwrite_if_in_dockerfile(tmpdir, docker_tasker, label_name, parent_val,
```

Comment:
I believe we are also missing a test case where both labels are present in the Dockerfile

Reply:
fixed